simple_salesforce/api.py

```diff
@@ -75,6 +78,10 @@ def __init__(
         self.sandbox = sandbox
         self.proxies = proxies
 
+        self.location = os.path.realpath(
+            os.path.join(os.getcwd(), os.path.dirname(__file__)))
```

Comment:
This will only work for test cases where the code's being executed in the project dir, not for dependent projects. That said, I'd personally prefer to see the templates inline in this file given it's really minimal.